apps/timetable/ui/js/index.js

```diff
@@ -72,6 +72,38 @@ define(['gh.core', 'gh.subheader', 'gh.calendar', 'gh.student-listview'], functi
     };
 
     /**
+     * Display the appropriate view depending on the state of the selected part
+     *
+     * @param  {Object}    ev      Standard jQuery event
+     * @param  {Object}    data
+     * @private
+     */
+    var onPartSelected = function(evt, data) {
+        if (!data.modules.results.length) {
+            gh.api.orgunitAPI.getOrgUnit(data.partId, true, function(err, data) {
+                gh.utils.renderTemplate($('#gh-empty-template'), {'data': data}, $('#gh-empty'));
+                $('#gh-left-container').css({'height': '215px', 'overflow': 'hidden'});
```

Comment:
Can you swap out the CSS with an `addClass/removeClass` so we keep the separation?